Makefile

```diff
@@ -344,9 +344,9 @@ endif
 
 .PHONY: yq
 yq: ## Download and install yq in local env
-	python -m venv $(LOCALBIN)/.virtualenv && \
-	source $(LOCALBIN)/.virtualenv/bin/activate && \
-	pip install yq
+	cd $(LOCALBIN) &&\
+	wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64.tar.gz -O - |\
+	tar xz && mv yq_linux_amd64 $(LOCALBIN)/yq
```

Comment:
```suggestion
	test -s $(LOCALBIN)/yq || GOBIN=$(LOCALBIN) github.com/mikefarah/yq/v4@latest
```

Reply:
I don't think this would work. The test is useful yes. But the `GOBIN=$(LOCALBIN) github.com/mikefarah/yq/v4@latest` doesn't actually invoke any command. I assume you meant `go install`, but bear in mind that the v4 of `yq` is using go v1.21. Whereas we are on v1.20.

That's bound to cause issues. By simply downloading the compiled binary we can avoid the compatibility issue, not to mention pulling more dependencies.

Reply:
i agree with that logic downloading the binary is preferable in this case.